examples/ConstraintIB/flow_past_cylinder_HF/HydroForceEval.cpp

```diff
@@ -55,9 +55,7 @@ template <typename T>
 std::string
 to_string(const T& n)
 {
-    std::ostringstream stm;
-    stm << n;
-    return stm.str();
+    return std::to_string(n);
 }
```

Comment:
We can probably just get rid of this function altogether.

Reply:
@drwells , so, delete lines 55 through 59?

Reply:
Yes. We should be able to just use `std::to_string` everywhere this was used.

Reply:
@drwells Ok, I think it's good now.